### ISSUE TITLE
close button colour contrast and focus visible

### ISSUE THREAD

**Issue by stef-barburou** [snap-framework/snap#14]
Language: English

Tool(s): WAT and colour contrast analyser

Browser/Screen reader: IE

WCAG 2.0 reference: 1.4.3 Contrast (minimum)
2.4.7 (Focus visible)
Details: AQA found that the close button for the help, sitemap and resources menus do not have a visible focus due to lack of colour contrast

![image](https://user-images.githubusercontent.com/41083520/74241575-51dfe380-4caa-11ea-900e-cf67b9a1a132.png)
